src/controllers/trainee/controller.ts

```diff
@@ -13,14 +13,22 @@ class TraineeController {
     userRepository: UserRepository = new UserRepository();
     get = async( req: Request, res: Response, next: NextFunction) => {
         try {
+           let traineecount = 0;
+           let { skip , limit , sort } = req.query;
+           const query = req.body;
+           sort = (sort === undefined || sort.length === 0 ) ? 'createdAt' : sort;
             console.log('Inside get function of Trainee Controller');
-          let resp = await this.userRepository.find({deletedAt: undefined}, {}, {});
-          console.log('response=', resp);
-          if (resp){
-                  console.log('Response of Repo is', resp);
-                    res.send({
-                    message: 'trainee fetch sucessfully',
-                    data: resp
+           const resp = await this.userRepository.find({deletedAt: undefined}, {}, { skip : Number(skip), limit : Number(limit), sort: { [String(sort)] : -1} });
+            if (resp) {
+                for (const users of resp) {
+                    if (users.role === 'trainee')
+                        traineecount++;
```

Comment:
The count should be the total no of records in the database, not the length of the current response array. So use separate query to read total number of records in DB (without skip and limit)